lioncore/metametamodel.adoc

```diff
@@ -124,10 +131,10 @@ It is indirectly a <<LanguageElement>> (as it is a top level element in a <<Lang
 
 [[Concept.extends, Concept.extends]]
 Each Concept `extends` zero or one Concepts.
-If no Concepts are explicitly extended, the Concept will implicitly extend the Concept named `BaseConcept`.
-`BaseConcept` is the only concept that truly does not extend any Concept.
+If no Concepts are explicitly extended, the Concept will implicitly extend the Concept named `Node`.
```

Comment:
Is Node both a class representing an instance and a Concept?

Reply:
We decided in https://github.com/LIonWeb-org/organization/issues/71 that the ultimate super-concept (akin to Java `Object` or MPS `BaseConcept`) is called `Node`.